source_gen/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
-## 1.5.1-wip
+## 2.0.0-wip
 
+- **Breaking Change**: Change `formatOutput` function to accept a language
+  version parameter.
+- **Formatting Change**: Generated code will no longer apply any fixes by
+  default (previously it would apply the single cascades statements fix). The
+  new formatter will not support applying fixes.
```

Comment:
Style nit: "will" -> "does". It's generally easier to read text in present tense, especially for people for whom English isn't their first language.